src/useWorkshopContext.ts

```diff
@@ -114,14 +124,44 @@ export function useWorkshopContext<T extends IConfigDefinition>(
 
   const insideIframe = isInsideIframe();
 
+  // Create a function to set the auto max height of the iframe (only works when Workshop widget height is set to "auto (max)")
+  const setAutoMaxHeight = React.useCallback((height: number) => {
+    if (isInsideIframe() && iframeWidgetId != null) {
+      sendMessageToWorkshop({
+        type: MESSAGE_TYPES_TO_WORKSHOP.SET_AUTO_MAX_HEIGHT,
+        iframeWidgetId,
+        height,
```

Comment:
should we do any validations? what happens if this number is a decimal or a negative number?